Geometry/HcalTowerAlgo/interface/HcalGeometry.h

```diff
@@ -104,6 +104,9 @@ class HcalGeometry : public CaloSubdetectorGeometry {
 			   CaloSubdetectorGeometry::IVec&   iVector,
 			   CaloSubdetectorGeometry::DimVec& dimVector,
 			   CaloSubdetectorGeometry::IVec& dinsVector ) const ;
+
+  const HcalTopology& topology() const { return theTopology; }
```

Comment:
It can be fixed in time. The whole geometry system likely needs some modernization or review in the near future.